docs/advanced/experimental/scaling.md

```diff
@@ -0,0 +1,60 @@
+# Scaling Executors in kubernetes
+
+In Jina we support to ways of scaling:
+- *Replicas* can be used with any Executor type.
+- *Shards* should only be used with Indexers, since they store a state.
+
+```{admonition} Important
+:class: important
+This page describes Jinas behavior with kubernetes deployment for now.
+When using `shards` and `replicas` without kubernetes, Jina will behave slightly different.
+It is discouraged to use the `parallel` argument, when deploying to kubernetes.
+```
+
+(replicas)=
+## Replicas
+
+Replication (or horizontal scaling) means duplicating an Executor and its state.
+It allows more requests to be served in parallel.
+A single request to Jina is only send to one of the replicas.
+Furthermore, it increases the failure tolerance.
+When one replica dies, another will immediately take over.
```

Comment:
But this is not true right now, is it? Do our replicas restart if they die? Or is this about k8s?